### DISCUSSION TITLE
Static heatmap

### DISCUSSION THREAD

**Discussion by Touville01** [roboflow/supervision#1315] [Q&A]
Hi everyone, 
II would like to know how I can generate a heatmap showing the most occupied areas (in terms of people passing through) over the entire duration of a video or live stream.

My goal is to create a heatmap that indicates the zones with the highest number of people passing through, whether it's for a period of 10 minutes, 1 hour, or even an entire day.

Do you have any advice or suggestions on the best way to implement this? Is this something that can be effectively done with sv.Heatmap, or do you have other recommendations?

Thanks

**Comment by LinasKo** [comment]
Hi @Touville01 👋 

You could run object detection on every frame, save the detections, and then merge them all with `sv.Detections.merge`.
If you heatmap-annotate the result, it may show the most visited areas.
